mitochondria-pipeline.inputs.json

```diff
@@ -0,0 +1,59 @@
+{
+  "##_COMMENT1": "Inputs",
+  "MitochondriaPipeline.wgs_aligned_input_bam": "gs://gatk-best-practices/mitochondria-pipeline/G97753.NA12878.bam",
```

Comment:
@schaluva noted that this variable could actually be a cram or bam. Not sure how best to rename it, but it could be `wgs_aligned_input` or `wgs_aligned_bam_or_cram`? Do we have a standard practice for when a task or tool is CRAM friendly?

Reply:
I'll do the replacement using `wgs_aligned_bam_or_cram`